meorg_client/cli.py

```diff
@@ -124,6 +124,19 @@ def file_upload(file_path):
         click.echo(f.get("file"))
 
 
+@click.command("upload_parallel")
+@click.argument("file_paths", nargs=-1)
+@click.option(
+    "-n", default=2, help="Number of simultaneous parallel uploads (default=2)."
+)
+def file_upload_parallel(file_paths, n=2):
+    client = _get_client()
+    responses = _call(client.upload_files_parallel, files=list(file_paths), n=n)
+    print(responses)
+    for response in responses:
+        click.echo(response.get("data").get("files")[0].get("file"))
```

Comment:
Why do we need to `echo` this specific part of the response schema here (is this for testing)?

Reply:
Oops, that print shouldn't be there! Will remove.

That response schema path is the actual file id that is used by other calls. The default behaviour is to return the ID of the file that was just uploaded to stdout so that this can be scripted in PBS.

```shell
$ meorg file upload file.txt
87y2345fgiuh
```

Reply:
That being said, I have a new helper function that pulls the ID out of the response object coming in another issue, as it is used half a dozen times.